### ISSUE TITLE
Filter Editor: fields are not in alphabetic order

### ISSUE THREAD

**Issue by guleri** [interactive-instruments/ldproxy#1262]
### ldproxy Version

3.6.1.

### Current Behavior

_I also posted this in the discussions section (#1255), but the difference between how this works when I specify the fields vs. when I use "include all", makes me believe that this isn't intended behavior. Something similar was also perceived as a bug/regression in #795._

I'm having some problems getting the filter fields in alphabetic order. That is, I'm having problems when I'm using "include all" like this:

    buildingBlock: QUERYABLES
    enabled: true
    included:
        "*"

If I specify what fields I want to include, e.g.:

included:
- vernetype
- navn

the filter fields are sorted.

### Expected Behavior

I expect that the filter fields are rendered in alphabetical order, regardless of if I specify the fields I want to be included or if I specify to include all existing fields.

### Steps to Reproduce

Adding the following in a config-file under defaults or for a specific service.
```
 - buildingBlock: QUERYABLES
   enabled: true
   included:
     - "*"
```

**Comment by guleri** [comment]
Looks like I was wrong about the filter fields being sorted if I specify fields. The filter fields order seems to be decided by the property order in the provider. Anyway, it seems like there's been a regression again (as mentioned in #795 and addressed in #848), so unless there's a workaround, I'll probably postpone the upgrade to v4.x, hoping it will be resolved.

**Comment by azahnen** [comment]
This will be fixed in v4.2.0.
